fe/src/components/Login.vue

```diff
@@ -34,12 +35,17 @@ export default {
   },
   methods: {
     submitForm() {
-      // not sure what to do here 
-      var res = httpService.post('api/user/create/', {email:'test@email.ca', password:'password'});
-      console.log(res)
-      console.log(res.data)
 
-    }  	
+      this.$refs['LoginForm'].validate((valid) => {
+        if(valid){
+          alert('Logging in')
+        }else{
+          alert('not logging in')
```

Comment:
can this be more descriptive? is there an error or another function parameter we can provide to give the user more information?

Reply:
Just had those in there for testing purposes...removed them now.